h2o-algos/src/test/java/hex/rulefit/RuleFitTest.java

```diff
@@ -128,7 +129,6 @@ public void testBestPracticeExampleWithLinearVariables() {
         try {
             Scope.enter();
             fr = parse_test_file("./smalldata/gbm_test/titanic.csv");
-            Scope.track(fr);
```

Comment:
I think you should tract the frame after all changes and then it should work correctly. 

Do you run this test on a multinode locally?

Reply:
Veronika is probably right, there are untracked vecs added. Haven't seen the failure. I'll try to come up with a sample fix.

Reply:
yes when I do only Scope.track even after changes it still fails on multinode (4 nodes)